python/aswfdocker/constants.py

```diff
@@ -32,4 +32,12 @@ class ImageType(enum.Enum):
 
 ALL = "all"
 
-CONAN_USER_HOME = "/tmp/c"
+# OS specific variables
+# Cross-building on windows hosts with linux docker should be possible so we keep both
+# variables available
+
+LINUX_CONAN_USER_HOME = "/tmp/c"
+WINDOWS_CONAN_USER_HOME = "C:\\tmp\\c"
```

Comment:
On Windows the system defined %TEMP% directory is typically `C:\Users\User Name\AppData\Local\Temp` aka `%USERPROFILE%\AppData\Local\Temp`, although I've seen a lot of local installs use `C:\Temp` which goes back to very old versions of Windows.